app/views/layouts/application.html.erb

```diff
@@ -36,7 +36,7 @@
         </ul>
         <ul class="nav justify-content-end">
           <li class="nav-item">
-            <a class="nav-link" href="<%=dashboard_path%>">Profile</a>
+            <a class="nav-link" href="<%=profile_path%>">Profile</a>
```

Comment:
Altered Nav-bar links to make more sense and work.

Reply:
Yeah that makes way more sense